name/templates/name/about.html

```diff
@@ -90,31 +88,39 @@
-    <a href="http://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection">302 Redirect</a> response:
-        <ul><li>user requests a record which has been merged with another record</li></ul>
-
-    <!--BEGIN PARAMETERS-->
-    <h3>How can I narrow my search?</h3>
-    Parameters hang off the end of a search request to narrow the results you get in your response and will get you closer to the name you are attempting to lookup.<br>
-    <b>q</b>: <i>Optional</i> - the text respresentation of the record name you want to search. Here's the two optional parameters available to you in a traditional search.<br>
-    <b>q_type</b>: <i>Optional</i> - the text represenation of the record type. 5 types are available to filter the search, "Personal", "Organization", "Event", "Building", "Software".<br>
-    <b>callback</b>: <i>Optional</i> - the text represenation of the jsonp callback wrapper.<br>
-
-    <!--  BEGIN LABEL API -->
-    <h3>Label API</h3>
-    The second method of searching is with the 'label' API. The label API is essentially a quick way to determine if an authorized name exists in the name app. The only two responses to this API are a 404 Not Found or a 302 Redirect.
-    <pre>
+                </pre>
+            </div>
+
+            <h3>Search return codes</h3>
+            <div>
+                <a href="http://en.wikipedia.org/wiki/List_of_HTTP_status_codes#2xx_Success">200 OK</a> response:
+                <ul><li>user gave a fully-formed search query which returns a list of results in JSON format</li></ul>
+
+                <a href="http://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error">410 Gone</a> response:
+                <ul><li>user gives a request for a record that had existed, but has since been deleted</li></ul>
+
+                <a href="http://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error">404 Not Found</a> response:
+                <ul><li>user gives a request for a record that does not exist</li></ul>
+
+                <a href="http://en.wikipedia.org/wiki/List_of_HTTP_status_codes#4xx_Client_Error">403 Forbidden</a> response:
+                <ul><li>user gives a request for a record that exits, but is non-public</li></ul>
+
+                <a href="http://en.wikipedia.org/wiki/List_of_HTTP_status_codes#3xx_Redirection">302 Redirect</a> response:
+                <ul><li>user requests a record which has been merged with another record</li></ul>
+            </div>
+
+            <!--BEGIN PARAMETERS-->
+            <h3>How can I narrow my search?</h3>
+
+            <p>Parameters hang off the end of a search request to narrow the results you get in your response and will get you closer to the name you are attempting to lookup.</p>
+
+            <strong>q</strong>: <em>Optional</em> - the text respresentation of the record name you want to search. Here the two optional parameters available to you in a traditional search.<br>
```

Comment:
Issue on "Here the two optional parameters available to you in a traditional search."